### PR TITLE
Development support for business scenarios

### PR DESCRIPTION
**Pull request by jrk94**
- Support for Business Scenarios
- Support for ESLint
- Support for Tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,21 +14,50 @@
...
+    "@typescript-eslint/eslint-plugin": "7.13.1",
+    "@typescript-eslint/parser": "7.13.1",
+    "chai-as-promised": "7.1.2",
+    "sinon": "^19.0.2",
     "typescript": "5.2.2"
   },
   "dependencies": {
     "@swc/core": "^1.5.7",
     "@vercel/ncc": "0.38.1",
+    "chai": "4.4.1",
```
Comment (oliveriosousa) [inline package.json]: should be moved to `devDependencies`

File: package.json [hunk 2]
```diff
@@ -14,21 +14,50 @@
...
+    "chai-spies": "1.1.0",
```
Comment (oliveriosousa) [inline package.json]: should be moved to `devDependencies`

File: package.json [hunk 3]
```diff
@@ -14,21 +14,50 @@
...
     "chalk": "5.3.0",
+    "eslint": "^8.53.0",
```
Comment (oliveriosousa) [inline package.json]: should be moved to `devDependencies`

File: package.json [hunk 4]
```diff
@@ -14,21 +14,50 @@
...
+    "eslint-plugin-mocha": "10.4.3",
```
Comment (oliveriosousa) [inline package.json]: should be moved to `devDependencies`

File: package.json [hunk 5]
```diff
@@ -14,21 +14,50 @@
...
     "fs-extra": "11.2.0",
     "inversify": "6.0.2",
+    "mocha": "10.4.0",
```
Comment (oliveriosousa) [inline package.json]: should be moved to `devDependencies`

File: package.json (hunk 2 above)
Comment (oliveriosousa) [reply package.json]: can't we use only `SinonJs` ?

**Review by rmmoreira84 — APPROVED** [review]
Great Work

**Pull request merged**